### PR TITLE
Dashboard Summary model

### PR DESCRIPTION
**Pull request by jinhyukchang**
### Summary of Changes

Adding Dashboard summary model

### Tests

N/A

### Documentation

N/A

### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [x] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
    - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [x] PR includes a summary of changes. 
- [x] PR adds unit tests, updates existing unit tests, __OR__ documents why no test additions or modifications are needed.
- [x] In case of new functionality, my PR adds documentation that describes how to use it.
    - All the public functions and the classes in the PR contain docstrings that explain what it does
- [x] PR passes `make test`

### CODE REVIEW

File: amundsen_common/models/dashboard.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+from typing import Optional
+
+import attr
+from marshmallow_annotations.ext.attrs import AttrsSchema
+
+
+@attr.s(auto_attribs=True, kw_only=True)
+class DashboardSummary:
+    uri: str = attr.ib()
```
Comment (danwom) [inline amundsen_common/models/dashboard.py:9]: What is the `uri` format? Is something like `{source}://{cluster}.{group_url}.{url}`?

Comment (jinhyukchang) [reply amundsen_common/models/dashboard.py:9]: Yes, it is something like:
`<product>_dashboard://<cluster>.<dashboard_group_id>/<dashboard_id>`

e.g: `mode_dashboard://gold.abcdefg/123456789`

**Comment by feng-tao** [comment]
@jinhyukchang is it due to github issue :) ? the pr is merged but we have the pr still in open state.

**Comment by jinhyukchang** [comment]
> @jinhyukchang is it due to github issue :) ? the pr is merged but we have the pr still in open state.

Yeah, I think I can close this one.

**Pull request closed without merging**